src/types/index.d.ts

```diff
@@ -328,3 +330,21 @@ export type MegaMenuPromoColumn = {
   link: MegaMenuLink
   description: string
 }
+
+/**
+ * Types for breadcrumb data
+ */
+export interface Breadcrumb {
```

Comment:
is this the type coming from json:api? if so i'd prefer it inside of `src/types/dataTypes/drupal/field_type.d.ts`

Reply:
Good call, moved